firmware/mcal/cli/periph/can.h

```diff
@@ -8,44 +8,37 @@
 #include <string.h>
```

Comment:
seems like this breaks the `stm32f767` platform, I think this should be fixed before this PR goes in